README.md

```diff
@@ -188,7 +188,7 @@ Required parameters:
 
 * name
 * periodicity
-* auth_key (if authentication is enabled in the server)
+* auth-key (if authentication is enabled in the server)
```

Comment:
The change to the cmcli usage is correct, but none of the other changes you have of auth_key -> auth-key are correct. The command-line option to cmcli is not the same as the REST parameters. For REST, it in fact needs to be auth_key, not-key. Please update the PR to only have the change to the cmcli usage and not all the other auth_key -> auth-key changes.